common/config/rush/command-line.json

```
@@ -31,6 +31,25 @@
       "shellCommand": "node common/scripts/ensure-consistent-versions.js",
       "safeForSimultaneousRushProcesses": true
     },
+    {
+      "commandKind": "bulk",
+      "name": "build:stable",
+      "summary": "Preprocess and build the GA version.",
```

nit: consistency with summarys; this says "build GA version" but below we say for "non-beta version".
```suggestion
      "summary": "Preprocess and build only stable (non-beta) code in packages",
```